conf/metafactory.lua

```diff
@@ -56,7 +56,8 @@ function basic(a)
             local set = mcs.set(prefix, num, 0, 30, size)
             mcs.write(set)
             mcs.flush()
-            local res = mcs.read()
+            local res = mcs.res_new()
```

Comment:
this code is commented and I don't really remember why.

for note though: Since this is an 'init' style mcshredder function (returns a function) the res can/should be pre-allocated with the other one above, ie:

`local set_res = mcs.res_new()` then use that down in this commented code instead of creating one to throw away

Reply:
Makes sense, updated.